src/Core/AdminConsole/Services/IEventMessageHandler.cs

```diff
@@ -5,4 +5,6 @@ namespace Bit.Core.Services;
 public interface IEventMessageHandler
 {
     Task HandleEventAsync(EventMessage eventMessage);
+
+    Task HandleManyEventAsync(IEnumerable<EventMessage> eventMessages);
```

Comment:
```suggestion
    Task HandleManyEventsAsync(IEnumerable<EventMessage> eventMessages);
```

🎨 Also check on multiple enumerations here -- sometimes it's more practical to just require a `List<>`.

Reply:
Made this change to the interface. I think we should be good on multiple enumerations - each handler should be able to do its work in a single pass.